### ISSUE TITLE
Suggestion: Provide example of how to use this using <Query> syntax

### ISSUE THREAD

**Issue by CaptainChemist** [dabit3/next-apollo-appsync#7]
I love this package but using the higher order component format is a little out of fashion by Apollo 2 standards. Could you provide an example of how you'd write this with the newer <Query> syntax? Thanks for the great package!

```
        <Query
          query={ALL_ITEMS_QUERY}
          fetchPolicy="cache-and-network"
          variables={{
            skip: this.props.page * perPage - perPage,
          }}
        >
          {({ data, error, loading }) => {
            if (loading) return <p>Loading...</p>;
            if (error) return <p>Error: {error.message}</p>;
            return (
              <ItemsList>{data.items.map(item => <Item item={item} key={item.id} />)}</ItemsList>
            );
          }}
        </Query>
```

**Comment by webberwang** [comment]
@CaptainChemist you can find more info on Apollo Client's docs

**Comment by CaptainChemist** [comment]
I ended up figuring it out. It was unclear to me initially that the `withData` only needs to be on the page level and all of the mutations and queries can be abstracted away in other files. I answered my question with the following repo that I made which has the same functionality as your repo but uses the new syntax. Hope that helps someone!
https://github.com/CaptainChemist/next-appsync-example/tree/newsyntax
